GeoLib/Raster.cpp

```diff
@@ -85,7 +85,8 @@ Raster* Raster::getRasterFromSurface(Surface const& sfc, double cell_size, doubl
         }
     }
 
-    RasterHeader header = {std::size_t(n_cols), std::size_t(n_rows), MathLib::Point3d(ll), cell_size, static_cast<double>(-9999)};
+    RasterHeader header;
+    header = {std::size_t(n_cols), std::size_t(n_rows), 1, MathLib::Point3d(ll), cell_size, static_cast<double>(-9999)};
```

Comment:
Why is the creation of the object 'header' and the initialization of the object in two steps now?

Reply:
No idea. Fixed it.